app/routes/users-index.vue

```diff
@@ -0,0 +1,43 @@
+<template lang="html">
+  <div class="">
+    <div class="section">
+      <div class="container">
+        <div class="users-header">
+          <h2>Users</h2>
+        </div>
+        <div class="grid">
+          <div class="grid__item" v-for="frog in users" v-bind:user="frog">
```

Comment:
Looks like you called the temporary looping variable `frog` but then used `user` on line 15.